src/ResourceManager/Compute/Commands.Compute/Strategies/StrategyCmdlet.cs

```diff
@@ -38,5 +46,24 @@ static class StrategyCmdlet
 
             return newState.Get(config) ?? current.Get(config);
         }
+
+        static string ToPowerShellString(object value)
+        {
+            if (value == null)
+            {
+                return "$null";
+            }
+            var s = value as string;
+            if (s != null)
+            {
+                return "\"" + s + "\"";
+            }
+            var e = value as IEnumerable;
+            if (e != null)
+            {
+                return string.Join(",", e.Cast<object>().Select(ToPowerShellString));
+            }
+            return value.ToString();
```

Comment:
so for this, if the type of the parameter (specifically for custom defined classes) does not implement a ToString() this would just print out the object type instead of a string representation of the data in the object? Do we want to do this?

Reply:
Yes, if we don't know the type we use `object.ToString()` function. This's also default PowerShell behavior.